concepts/05 UI Components/Lookup/00 Getting Started with Lookup/35 Enable Pagination.md

```diff
@@ -0,0 +1,89 @@
+You can enable pagination of the Lookup component. This feature is intended to render the Lookup items. If the item data is remote, pagination allows to load items by parts to speed up the Lookup's working process. Follow the steps below to enable this functionality:
```

Comment:
```suggestion
Pagination allows Lookup to *render* items in batches. When data is remote, the component can also *load* items in batches if the server can partition data. Follow the steps below to enable this functionality:
```

"Allows to" is incorrect. [More on this](https://www.macmillandictionary.com/dictionary/british/allow)